### ISSUE TITLE
After signing in, user should return to page they were previously on

### ISSUE THREAD

**Issue by lisahackett1978** [RecordReplay/devtools#1903]
**Description:**
When logged out of the Replay browser and attempting to add a comment, Replay prompts the user to log in. After logging in user always arrives at the recordings page, not the screen they were on previously.

**Steps to Reproduce:**
1. Log out of Replay
2. Quit Replay
3. Open Replay, copy URL of a recording into the address bar, go
4. Attempt to add a comment
5. Log in
6. Arrive at recordings page

**Expected Result:**
After signing in, return user to page they were previously on
**Additional Context:**

* Screenshots:
* Video (Loom):
* Replay:

**Comment by jonbell-lot23** [comment]
100%

**Comment by jasonLaster** [comment]
Thanks Lisa for the writeup. I had filed another issue #1712, which was a duplicate but your version is clearer.

My hope is that we can fix this when we change Authentication providers. We're currently exploring clerk.dev and AWS cognito.

**Comment by hbenl** [comment]
We can fix this without changing our auth provider, see https://github.com/RecordReplay/devtools/issues/1761#issuecomment-792765142

**Comment by hbenl** [comment]
depends on #2062
